--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -37,6 +37,7 @@
 #include "chrome/browser/apps/app_discovery_service/app_discovery_service.h"
 #include "chrome/browser/browser_features.h"
 #include "chrome/browser/browser_process.h"
+#include "chrome/browser/channel_selection.h"
 #include "chrome/browser/contextual_cueing/features.h"
 #include "chrome/browser/default_browser/default_browser_features.h"
 #include "chrome/browser/devtools/features.h"
--- /dev/null
+++ b/chrome/browser/channel_selection.cc
@@ -0,0 +1,28 @@
+// Copyright 2025 The Helium Authors
+// You can use, redistribute, and/or modify this source code under
+// the terms of the GPL-3.0 license that can be found in the LICENSE file.
+
+#include <string>
+
+#include "base/command_line.h"
+#include "chrome/browser/channel_selection.h"
+
+namespace helium {
+
+std::string GetSelectedChannel() {
+  auto* command_line = base::CommandLine::ForCurrentProcess();
+
+  if (command_line->HasSwitch(kChannelCommandLine)) {
+    std::string override_channel =
+        command_line->GetSwitchValueASCII(kChannelCommandLine);
+
+    if ((override_channel == kChannelStable) ||
+        (override_channel == kChannelBeta)) {
+      return override_channel;
+    }
+  }
+
+  return kChannelStable;
+}
+
+}  // namespace helium
--- /dev/null
+++ b/chrome/browser/channel_selection.h
@@ -0,0 +1,19 @@
+// Copyright 2025 The Helium Authors
+// You can use, redistribute, and/or modify this source code under
+// the terms of the GPL-3.0 license that can be found in the LICENSE file.
+
+#ifndef CHROME_BROWSER_CHANNEL_SELECTION_H_
+#define CHROME_BROWSER_CHANNEL_SELECTION_H_
+
+#include <string>
+namespace helium {
+
+constexpr const char kChannelCommandLine[] = "helium-update-channel";
+
+constexpr const char kChannelStable[] = "stable";
+constexpr const char kChannelBeta[] = "beta";
+
+std::string GetSelectedChannel();
+
+}  // namespace helium
+#endif  /* CHROME_BROWSER_CHANNEL_SELECTION_H_ */
--- a/chrome/browser/helium_flag_choices.h
+++ b/chrome/browser/helium_flag_choices.h
@@ -21,6 +21,12 @@ constexpr const FeatureEntry::Choice kEn
     {"Disabled", kEnergySaverFrameRateCommandLine, "9001"},
 };
 
+constexpr const FeatureEntry::Choice kChannelChoices[] = {
+    {flags_ui::kGenericExperimentChoiceAutomatic, "", ""},
+    {kChannelStable, kChannelCommandLine, kChannelStable},
+    {kChannelBeta, kChannelCommandLine, kChannelBeta},
+};
+
 }  // namespace helium
 
 #endif /* CHROME_BROWSER_HELIUM_FLAG_CHOICES_H_ */
--- a/chrome/browser/helium_flag_entries.h
+++ b/chrome/browser/helium_flag_entries.h
@@ -16,6 +16,9 @@
      "Controls whether Helium asks for confirmation before opening 15 or more "
      "bookmarks at once. Helium flag.",
      kOsDesktop, FEATURE_VALUE_TYPE(features::kBookmarkOpenAllConfirmation)},
+    {helium::kChannelCommandLine, "Update channel",
+     "Selects which update channel to use for update checking. Helium flag.",
+     kOsAll, MULTI_VALUE_TYPE(helium::kChannelChoices)},
     // core/network
     // ui
     // platform
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -286,6 +286,12 @@ source_set("chained_back_navigation_trac
   ]
 }
 
+source_set("channel_selection") {
+  public = [ "channel_selection.h" ]
+  sources = [ "channel_selection.cc" ]
+  deps = [ "//base" ]
+}
+
 source_set("child_process_host_flags") {
   public = [ "child_process_host_flags.h" ]
   deps = [ "//content/public/browser" ]
@@ -894,6 +900,7 @@ source_set("core") {
     ":browser_process_platform_part",
     ":browser_public_dependencies",
     ":chained_back_navigation_tracker",
+    ":channel_selection",
     ":child_process_host_flags",
     ":chrome_browser_field_trials",
     ":chrome_browser_interface_binders_webui_parts",
