--- a/chrome/browser/ui/views/frame/system_menu_model_builder.cc
+++ b/chrome/browser/ui/views/frame/system_menu_model_builder.cc
@@ -161,6 +161,13 @@ void SystemMenuModelBuilder::BuildSystem
   AddItemWithIconMaybe(model, IDC_NAME_WINDOW, IDS_NAME_WINDOW,
                        kNameWindowOldIcon);
 #endif  // BUILDFLAG(IS_MAC)
+
+  if (!browser()->GetProfile()->IsOffTheRecord()) {
+    model->AddSeparator(ui::NORMAL_SEPARATOR);
+    model->AddItemWithStringId(IDC_SHOW_CUSTOMIZE_CHROME_SIDE_PANEL,
+                               IDS_SHOW_CUSTOMIZE_CHROME_SIDE_PANEL);
+  }
+
   model->AddSeparator(ui::NORMAL_SEPARATOR);
   model->AddItemWithStringId(IDC_TAB_SEARCH_TOGGLE_PIN,
                              IDS_TAB_STRIP_PIN_TAB_SEARCH);
