--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
-COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+COMPANY_FULLNAME=The Helium Authors
+COMPANY_SHORTNAME=The Helium Authors
+PRODUCT_FULLNAME=Helium
+PRODUCT_SHORTNAME=Helium
+PRODUCT_INSTALLER_FULLNAME=Helium Installer
+PRODUCT_INSTALLER_SHORTNAME=Helium Installer
+COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Helium Authors. All rights reserved.
+MAC_BUNDLE_ID=net.imput.helium
 MAC_CREATOR_CODE=Cr24
-MAC_TEAM_ID=
+MAC_TEAM_ID=S4Q33XPHB4
--- a/chrome/app/chrome_crash_reporter_client.cc
+++ b/chrome/app/chrome_crash_reporter_client.cc
@@ -129,25 +129,7 @@ bool ChromeCrashReporterClient::GetCrash
 
 void ChromeCrashReporterClient::GetProductInfo(ProductInfo* product_info) {
   CHECK(product_info);
-
-#if BUILDFLAG(IS_ANDROID)
-  product_info->product_name = "Chrome_Android";
-#elif BUILDFLAG(IS_CHROMEOS)
-  product_info->product_name = "Chrome_ChromeOS";
-#elif BUILDFLAG(IS_LINUX)
-#if defined(ADDRESS_SANITIZER)
-  product_info->product_name = "Chrome_Linux_ASan";
-#else
-  product_info->product_name = "Chrome_Linux";
-#endif  // defined(ADDRESS_SANITIZER)
-#elif BUILDFLAG(IS_MAC)
-  product_info->product_name = "Chrome_Mac";
-#elif BUILDFLAG(IS_WIN)
-  product_info->product_name = "Chrome";
-#else
-  NOTREACHED();
-#endif
-
+  product_info->product_name = "Helium";
   product_info->version = UpdaterVersion();
   product_info->channel =
       chrome::GetChannelName(chrome::WithExtendedStable(true));
