--- a/chrome/browser/ui/omnibox/omnibox_pedal_implementations.cc
+++ b/chrome/browser/ui/omnibox/omnibox_pedal_implementations.cc
@@ -2039,12 +2039,7 @@ GetPedalImplementations(bool incognito,
   if (!incognito && !guest) {
     add(new OmniboxPedalClearBrowsingData(/*incognito=*/false));
   }
-  add(new OmniboxPedalManagePasswords());
-  add(new OmniboxPedalUpdateCreditCard());
   add(new OmniboxPedalLaunchIncognito());
-  if (!base::android::device_info::is_automotive()) {
-    add(new OmniboxPedalRunChromeSafetyCheck());
-  }
   add(new OmniboxPedalPlayChromeDinoGame());
   add(new OmniboxPedalManageSiteSettings());
   add(new OmniboxPedalManageChromeSettings());
@@ -2056,37 +2051,15 @@ GetPedalImplementations(bool incognito,
   if (!guest) {
     add(new OmniboxPedalClearBrowsingData(incognito));
   }
-  add(new OmniboxPedalManagePasswords());
-  add(new OmniboxPedalUpdateCreditCard());
   add(new OmniboxPedalLaunchIncognito());
-  add(new OmniboxPedalTranslate());
   add(new OmniboxPedalUpdateChrome());
-  add(new OmniboxPedalRunChromeSafetyCheck());
   add(new OmniboxPedalManageSecuritySettings());
   add(new OmniboxPedalManageCookies());
-  add(new OmniboxPedalManageAddresses());
-  add(new OmniboxPedalManageSync());
   add(new OmniboxPedalManageSiteSettings());
-  add(new OmniboxPedalSeeChromeTips());
-
-  if (testing || BUILDFLAG(GOOGLE_CHROME_BRANDING)) {
-    add(new OmniboxPedalCreateGoogleDoc());
-    add(new OmniboxPedalCreateGoogleSheet());
-    add(new OmniboxPedalCreateGoogleSlide());
-    add(new OmniboxPedalCreateGoogleCalendarEvent());
-    add(new OmniboxPedalCreateGoogleSite());
-    add(new OmniboxPedalCreateGoogleKeepNote());
-    add(new OmniboxPedalCreateGoogleForm());
-    add(new OmniboxPedalManageGoogleAccount());
-    add(new OmniboxPedalChangeGooglePassword());
-  }
-
   if (incognito) {
     add(new OmniboxPedalCloseIncognitoWindows());
   }
   add(new OmniboxPedalPlayChromeDinoGame());
-  add(new OmniboxPedalFindMyPhone());
-  add(new OmniboxPedalManageGooglePrivacy());
   add(new OmniboxPedalManageChromeSettings());
   add(new OmniboxPedalManageChromeDownloads());
   add(new OmniboxPedalViewChromeHistory());
