--- a/chrome/browser/ui/ui_features.cc
+++ b/chrome/browser/ui/ui_features.cc
@@ -252,13 +252,7 @@ BASE_FEATURE(kPasswordSaveUpdateDropdown
              base::FEATURE_DISABLED_BY_DEFAULT);
 #endif
 
-BASE_FEATURE(kSidePanelFlyoverAnimation,
-#if BUILDFLAG(IS_MAC)
-             base::FEATURE_DISABLED_BY_DEFAULT
-#else
-             base::FEATURE_ENABLED_BY_DEFAULT
-#endif
-);
+BASE_FEATURE(kSidePanelFlyoverAnimation, base::FEATURE_DISABLED_BY_DEFAULT);
 
 bool UseSidePanelFlyoverAnimation() {
   return base::FeatureList::IsEnabled(kSidePanelFlyoverAnimation);
