--- a/components/bookmarks/browser/bookmark_utils.cc
+++ b/components/bookmarks/browser/bookmark_utils.cc
@@ -451,7 +451,7 @@ bool DoesBookmarkContainWords(const std:
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
-      prefs::kShowBookmarkBar, true,
+      prefs::kShowBookmarkBar, false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
   // `BookmarkBarVisibilityState::kOnlyShowOnNtp` is equivalent to
   // `kShowBookmarkBar` set to false.
