--- a/components/extstore_fixups/BUILD.gn
+++ b/components/extstore_fixups/BUILD.gn
@@ -16,6 +16,7 @@ generate_grd("extstore_fixups_grd") {
     input_files_base_dir = rebase_path(".", "//")
     input_files = [
         "manifest.json",
+        "chrome-webstore.js",
     ]
 }
 
--- a/components/extstore_fixups/manifest.json
+++ b/components/extstore_fixups/manifest.json
@@ -9,5 +9,10 @@
   "version": "0.0.0",
   "manifest_version": 3,
   "content_scripts": [
+    {
+      "matches": [ "https://chromewebstore.google.com/*" ],
+      "run_at": "document_start",
+      "js": [ "chrome-webstore.js" ]
+    }
   ]
 }
--- /dev/null
+++ b/components/extstore_fixups/chrome-webstore.js
@@ -0,0 +1,74 @@
+// Copyright 2025 The Helium Authors
+// You can use, redistribute, and/or modify this source code under
+// the terms of the GPL-3.0 license that can be found in the LICENSE file.
+
+(() => {
+  const PROMO_POPUP_SELECTOR = 'header > [aria-labelledby=promo-header]';
+  const SWITCH_NAG_SELECTOR = 'main section [aria-label=Info]';
+  const SWITCH_BTN_SELECTOR = 'button[aria-label^=\'Switch to Chrome\']';
+  const removeChromeNags = () => {
+    const promoPopup = document.querySelector(PROMO_POPUP_SELECTOR);
+    if (promoPopup) {
+      promoPopup.remove();
+    }
+
+    const switchNag = document.querySelector(SWITCH_NAG_SELECTOR);
+    if (switchNag && switchNag.querySelector(SWITCH_BTN_SELECTOR)) {
+      switchNag.remove();
+    }
+  };
+
+  const replaceDownloadButtonText = (el) => {
+    if (el.nodeType === Node.TEXT_NODE) {
+      if (!el.nodeValue?.includes('Chrome')) {
+        return false;
+      }
+
+      el.nodeValue = el.nodeValue.replace(/\b(Google\s)?Chrome\b/, 'Helium');
+      return true;
+    } else if (el.childNodes) {
+      for (const node of el.childNodes) {
+        if (replaceDownloadButtonText(node)) {
+          return true;
+        }
+      }
+    }
+
+    return false;
+  };
+
+  const getDownloadButton = (section) => {
+    const buttons = section?.querySelectorAll('button') || [];
+
+    for (const button of buttons) {
+        if (!button.textContent?.includes('Chrome')) {
+            continue;
+        }
+
+        return button;
+    }
+  }
+
+  const changeDownloadButtonText = () => {
+    const mainTitles = document.querySelectorAll('main section h1');
+
+    for (const mainTitleEl of mainTitles) {
+      const downloadButton = getDownloadButton(mainTitleEl?.parentElement?.parentElement);
+      if (!downloadButton) {
+        continue;
+      }
+
+      if (replaceDownloadButtonText(downloadButton)) {
+        break;
+      }
+    }
+  };
+
+  const handleChanges = () => {
+    removeChromeNags();
+    changeDownloadButtonText();
+  };
+
+  const observer = new MutationObserver(handleChanges);
+  observer.observe(document, {childList: true, subtree: true});
+})();
