--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -432,11 +432,13 @@ namespace about_flags {
 
 namespace {
 
-const unsigned kOsAll = kOsMac | kOsWin | kOsLinux | kOsCrOS | kOsAndroid;
-const unsigned kOsDesktop = kOsMac | kOsWin | kOsLinux | kOsCrOS;
+const unsigned kOsAll = kOsMac | kOsWin | kOsLinux;
+const unsigned kOsDesktop = kOsMac | kOsWin | kOsLinux;
+
+constexpr char kHeliumFlagFormattingSentinel[] = "dummy-flag-for-formatting";
 
 #if defined(USE_AURA)
-const unsigned kOsAura = kOsWin | kOsLinux | kOsCrOS;
+const unsigned kOsAura = kOsWin | kOsLinux;
 #endif  // USE_AURA
 
 #if defined(USE_AURA)
@@ -4937,11 +4939,13 @@ constexpr char kEnterpriseIsolatedModeIn
     "force-enterprise-isolated-mode";
 
 // When adding a new choice, add it to the end of the list.
+#include "chrome/browser/helium_flag_choices.h"
 #include "chrome/browser/ungoogled_flag_choices.h"
 #include "chrome/browser/bromite_flag_choices.h"
 #include "chrome/browser/ungoogled_platform_flag_choices.h"
 #include "chrome/browser/existing_switch_flag_choices.h"
 const FeatureEntry kFeatureEntries[] = {
+#include "chrome/browser/helium_flag_entries.h"
 #include "chrome/browser/ungoogled_flag_entries.h"
 #include "chrome/browser/bromite_flag_entries.h"
 #include "chrome/browser/ungoogled_platform_flag_entries.h"
@@ -13949,6 +13953,10 @@ void GetStorage(Profile* profile, GetSto
 
 bool ShouldSkipConditionalFeatureEntry(const flags_ui::FlagsStorage* storage,
                                        const FeatureEntry& entry) {
+  if (std::string_view(entry.internal_name) == kHeliumFlagFormattingSentinel) {
+    return true;
+  }
+
   version_info::Channel channel = chrome::GetChannel();
 #if BUILDFLAG(IS_CHROMEOS)
   // enable-ui-devtools is only available on for non Stable channels.
--- /dev/null
+++ b/chrome/browser/helium_flag_choices.h
@@ -0,0 +1,16 @@
+// Copyright 2025 The Helium Authors
+// You can use, redistribute, and/or modify this source code under
+// the terms of the GPL-3.0 license that can be found in the LICENSE file.
+
+#ifndef CHROME_BROWSER_HELIUM_FLAG_CHOICES_H_
+#define CHROME_BROWSER_HELIUM_FLAG_CHOICES_H_
+
+#include "chrome/browser/channel_selection.h"
+#include "components/webui/flags/feature_entry.h"
+
+namespace helium {
+using namespace ::helium;
+
+}  // namespace helium
+
+#endif /* CHROME_BROWSER_HELIUM_FLAG_CHOICES_H_ */
--- /dev/null
+++ b/chrome/browser/helium_flag_entries.h
@@ -0,0 +1,13 @@
+// Copyright 2025 The Helium Authors
+// You can use, redistribute, and/or modify this source code under
+// the terms of the GPL-3.0 license that can be found in the LICENSE file.
+
+#ifndef CHROME_BROWSER_HELIUM_FLAG_ENTRIES_H_
+#define CHROME_BROWSER_HELIUM_FLAG_ENTRIES_H_
+{kHeliumFlagFormattingSentinel, "", "", flags_ui::kFlagInfrastructure,
+ SINGLE_VALUE_TYPE("")},
+    // core
+    // core/network
+    // ui
+    // platform
+#endif  // CHROME_BROWSER_HELIUM_FLAG_ENTRIES_H_
