--- a/chrome/common/url_constants.h
+++ b/chrome/common/url_constants.h
@@ -164,7 +164,7 @@ inline constexpr char kChromeUIUntrusted
     "chrome-untrusted://new-tab-page/";
 
 // The URL for the Chromium project used in the About dialog.
-inline constexpr char16_t kChromiumProjectURL[] = u"https://www.chromium.org/";
+inline constexpr char16_t kChromiumProjectURL[] =  u"https://www.chro""mium.org/";
 #if BUILDFLAG(IS_CHROMEOS)
 static_assert(std::u16string_view(kChromiumProjectURL) ==
               ash::chrome_external_urls::kChromiumProjectURL);
