--- a/chrome/browser/ui/browser_ui_prefs.cc
+++ b/chrome/browser/ui/browser_ui_prefs.cc
@@ -75,7 +75,7 @@ void RegisterBrowserPrefs(PrefRegistrySi
 
   registry->RegisterBooleanPref(prefs::kHoverCardImagesEnabled, true);
 
-  registry->RegisterBooleanPref(prefs::kHoverCardMemoryUsageEnabled, true);
+  registry->RegisterBooleanPref(prefs::kHoverCardMemoryUsageEnabled, false);
 
   registry->RegisterBooleanPref(
       prefs::kHoverCardMemoryUsageDisableMigrationComplete, false);
@@ -174,7 +174,7 @@ void RegisterBrowserUserPrefs(user_prefs
 
   // We need to register the type of these preferences in order to query
   // them even though they're only typically controlled via policy.
-  registry->RegisterBooleanPref(policy::policy_prefs::kHideWebStoreIcon, false);
+  registry->RegisterBooleanPref(policy::policy_prefs::kHideWebStoreIcon, true);
 
 #if BUILDFLAG(IS_MAC)
   // This really belongs in platform code, but there's no good place to
