--- a/chrome/chrome_repack_locales.gni
+++ b/chrome/chrome_repack_locales.gni
@@ -106,12 +106,6 @@ template("chrome_repack_locales") {
           [ "${root_gen_dir}/chrome/browser/indigo/resources/indigo_strings_" ]
       deps += [ "//chrome/browser/indigo/resources:indigo_strings" ]
     }
-    if (enable_builtin_search_provider_assets && !is_android) {
-      source_patterns += [ "${root_gen_dir}/third_party/search_engines_data/search_engine_descriptions_strings_" ]
-      deps += [
-        "//third_party/search_engines_data:search_engine_descriptions_strings",
-      ]
-    }
     if (enable_extensions_core) {
       source_patterns +=
           [ "${root_gen_dir}/extensions/strings/extensions_strings_" ]
--- a/components/search_engines/template_url.cc
+++ b/components/search_engines/template_url.cc
@@ -60,10 +60,6 @@
 #include "ui/base/l10n/l10n_util.h"
 #include "url/gurl.h"
 
-#if BUILDFLAG(ENABLE_BUILTIN_SEARCH_PROVIDER_ASSETS) && !BUILDFLAG(IS_ANDROID)
-#include "third_party/search_engines_data/search_engine_descriptions_strings_map.h"
-#endif
-
 namespace {
 
 const bool kEnableBuiltinSearchProviderAssets =
@@ -2015,18 +2011,6 @@ std::string TemplateURL::GetBuiltinDescr
 }
 
 std::optional<std::u16string> TemplateURL::GetBuiltinMarketingSnippet() const {
-#if BUILDFLAG(ENABLE_BUILTIN_SEARCH_PROVIDER_ASSETS) && !BUILDFLAG(IS_ANDROID)
-  auto resource_id = GetBuiltinDescriptionResourceId();
-  if (!resource_id.empty()) {
-    auto iter = std::ranges::find_if(
-        kSearchEngineDescriptionsStrings,
-        [&](const auto& resource) { return resource.path == resource_id; });
-
-    if (iter != std::end(kSearchEngineDescriptionsStrings)) {
-      return l10n_util::GetStringUTF16(iter->id);
-    }
-  }
-#endif
   return std::nullopt;
 }
 
--- a/third_party/search_engines_data/BUILD.gn
+++ b/third_party/search_engines_data/BUILD.gn
@@ -40,10 +40,6 @@ if (enable_builtin_search_provider_asset
 
   group("resources") {
     public_deps = [ ":search_engines_scaled_resources" ]
-
-    if (!is_android) {
-      public_deps += [ ":search_engine_descriptions_strings" ]
-    }
   }
 
   grit("search_engines_scaled_resources") {
@@ -62,18 +58,4 @@ if (enable_builtin_search_provider_asset
     ]
     output_dir = target_gen_dir
   }
-
-  if (!is_android) {
-    grit_strings("search_engine_descriptions_strings") {
-      source = "resources_internal/search_engine_descriptions_strings.grd"
-      inputs = rebase_path(search_engine_description_xtb_files,
-                           ".",
-                           "resources_internal")
-      outputs = [
-        "search_engine_descriptions_strings.h",
-        "search_engine_descriptions_strings_map.cc",
-        "search_engine_descriptions_strings_map.h",
-      ]
-    }
-  }
 }
