--- a/chrome/browser/ui/toolbar/app_menu_model.cc
+++ b/chrome/browser/ui/toolbar/app_menu_model.cc
@@ -251,11 +251,6 @@ void AddSubMenuWithStringIdAndVectorIcon
                                      ui::SimpleMenuModel::kDefaultIconSize));
 }
 
-const gfx::VectorIcon& GetSendTabToSelfIcon() {
-  return features::IsRoundedIconsEnabled() ? kDevicesIcon
-                                           : kDevicesChromeRefreshOldIcon;
-}
-
 // Conditionally return the update app menu item title based on upgrade detector
 // state.
 std::u16string GetUpgradeDialogTitleText() {
@@ -820,50 +815,8 @@ class SaveAndShareSubMenuModel : public
   SaveAndShareSubMenuModel& operator=(const SaveAndShareSubMenuModel&) = delete;
   ~SaveAndShareSubMenuModel() override = default;
 
- private:
-  // Builds Send Tab to Self target device submenu when enhanced desktop UI is
-  // enabled.
-  void BuildSendTabToSelfSubmenu(BrowserWindowInterface* browser,
-                                 content::WebContents* web_contents);
-
-  // Fallback helper to add simple Send Tab to Self menu item.
-  void BuildSendTabToSelfSimpleItem();
-
-  std::unique_ptr<send_tab_to_self::SendTabToSelfContextMenuDelegate>
-      send_tab_to_self_submenu_delegate_;
-  std::unique_ptr<ui::SimpleMenuModel> send_tab_to_self_submenu_;
 };
 
-void SaveAndShareSubMenuModel::BuildSendTabToSelfSubmenu(
-    BrowserWindowInterface* browser,
-    content::WebContents* web_contents) {
-  CHECK(web_contents);
-
-  send_tab_to_self_submenu_delegate_ =
-      std::make_unique<send_tab_to_self::SendTabToSelfContextMenuDelegate>(
-          web_contents, send_tab_to_self::ShareEntryPoint::kShareMenu);
-  send_tab_to_self_submenu_ = std::make_unique<ui::SimpleMenuModel>(
-      send_tab_to_self_submenu_delegate_.get());
-  send_tab_to_self_submenu_delegate_->PopulateSubmenu(
-      send_tab_to_self_submenu_.get());
-
-  AddSubMenuWithStringIdAndIcon(
-      IDC_SEND_TAB_TO_SELF, IDS_MENU_SEND_TAB_TO_SELF,
-      send_tab_to_self_submenu_.get(),
-      ui::ImageModel::FromVectorIcon(GetSendTabToSelfIcon()));
-
-  SetIsNewFeatureAt(GetItemCount() - 1,
-                    UserEducationService::MaybeShowNewBadge(
-                        browser->GetProfile(),
-                        send_tab_to_self::kSendTabToSelfEnhancedDesktopUIv2));
-}
-
-void SaveAndShareSubMenuModel::BuildSendTabToSelfSimpleItem() {
-  AddItemWithStringIdAndVectorIcon(this, IDC_SEND_TAB_TO_SELF,
-                                   IDS_MENU_SEND_TAB_TO_SELF,
-                                   GetSendTabToSelfIcon());
-}
-
 SaveAndShareSubMenuModel::SaveAndShareSubMenuModel(
     ui::SimpleMenuModel::Delegate* delegate,
     BrowserWindowInterface* browser)
@@ -911,27 +864,6 @@ SaveAndShareSubMenuModel::SaveAndShareSu
           features::IsRoundedIconsEnabled() ? vector_icons::kLinkIcon
                                             : kLinkChromeRefreshOldIcon);
 
-      // WebContents is required to query target devices and display state for
-      // Send Tab to Self.
-      content::WebContents* web_contents =
-          browser->tab_strip_model()->GetActiveWebContents();
-      std::optional<send_tab_to_self::EntryPointDisplayReason> reason =
-          web_contents
-              ? send_tab_to_self::GetEntryPointDisplayReason(web_contents)
-              : std::nullopt;
-
-      // When enhanced desktop UI v2 feature flag is enabled and feature is
-      // offered, build submenu with available devices. Otherwise, build simple
-      // command item.
-      if (web_contents &&
-          base::FeatureList::IsEnabled(
-              send_tab_to_self::kSendTabToSelfEnhancedDesktopUIv2) &&
-          reason == send_tab_to_self::EntryPointDisplayReason::kOfferFeature) {
-        BuildSendTabToSelfSubmenu(browser, web_contents);
-      } else {
-        BuildSendTabToSelfSimpleItem();
-      }
-
       AddItemWithStringIdAndVectorIcon(
           this, IDC_QRCODE_GENERATOR, IDS_APP_MENU_CREATE_QR_CODE,
           features::IsRoundedIconsEnabled() ? kQrCodeIcon
@@ -946,11 +878,6 @@ SaveAndShareSubMenuModel::SaveAndShareSu
   }
 }
 
-bool ArePromotionsEnabled() {
-  PrefService* local_state = g_browser_process->local_state();
-  return local_state && local_state->GetBoolean(prefs::kPromotionsEnabled);
-}
-
 }  // namespace
 
 ////////////////////////////////////////////////////////////////////////////////
@@ -1068,55 +995,6 @@ void ToolsMenuModel::Build(BrowserWindow
       this, IDC_NAME_WINDOW, IDS_NAME_WINDOW,
       features::IsRoundedIconsEnabled() ? kWebAssetIcon : kNameWindowOldIcon);
 
-  if (auto* controller = tabs::VerticalTabStripStateController::From(browser)) {
-    if (controller->ShouldDisplayVerticalTabs()) {
-      AddItemWithStringIdAndVectorIcon(
-          this, IDC_TOGGLE_VERTICAL_TABS, IDS_SWITCH_TO_HORIZONTAL_TAB,
-          features::IsRoundedIconsEnabled() ? kToolbarIcon : kToolbarOldIcon);
-    } else {
-      AddItemWithStringIdAndVectorIcon(
-          this, IDC_TOGGLE_VERTICAL_TABS, IDS_SWITCH_TO_VERTICAL_TAB,
-          base::i18n::IsRTL() ? features::IsRoundedIconsEnabled()
-                                    ? kDockToLeftIcon
-                                    : kDockToRightOldIcon
-          : features::IsRoundedIconsEnabled() ? kDockToRightIcon
-                                              : kDockToLeftOldIcon);
-      const bool use_preview_badge =
-          base::FeatureList::IsEnabled(tabs::kVerticalTabsPreviewBadge);
-      const ui::NewBadgeType badge_type = use_preview_badge
-                                              ? ui::NewBadgeType::kPreview
-                                              : ui::NewBadgeType::kNew;
-      const user_education::DisplayNewBadge show_badge =
-          UserEducationService::MaybeShowNewBadge(
-              browser->GetProfile(), use_preview_badge
-                                         ? tabs::kVerticalTabsPreviewBadge
-                                         : tabs::kVerticalTabsNewBadge);
-      SetIsNewFeatureAt(GetIndexOfCommandId(IDC_TOGGLE_VERTICAL_TABS).value(),
-                        show_badge, badge_type);
-    }
-  }
-
-  if (CustomizeChromePageHandler::IsSupported(
-          NtpCustomBackgroundServiceFactory::GetForProfile(
-              browser->GetProfile()),
-          browser->GetProfile())) {
-    AddItemWithStringIdAndVectorIcon(this, IDC_SHOW_CUSTOMIZE_CHROME_SIDE_PANEL,
-                                     IDS_SHOW_CUSTOMIZE_CHROME_SIDE_PANEL,
-                                     features::IsRoundedIconsEnabled()
-                                         ? kEditIcon
-                                         : kEditChromeRefreshOldIcon);
-  }
-
-  AddSeparator(ui::NORMAL_SEPARATOR);
-
-  AddItemWithStringIdAndVectorIcon(
-      this, IDC_SHOW_READING_MODE_SIDE_PANEL, IDS_SHOW_READING_MODE_SIDE_PANEL,
-      features::IsRoundedIconsEnabled() ? kMenuBookIcon
-                                        : kMenuBookChromeRefreshOldIcon);
-  SetElementIdentifierAt(
-      GetIndexOfCommandId(IDC_SHOW_READING_MODE_SIDE_PANEL).value(),
-      kReadingModeMenuItem);
-
   AddSeparator(ui::NORMAL_SEPARATOR);
 
   AddItemWithStringIdAndVectorIcon(
@@ -1163,6 +1041,12 @@ void ToolsMenuModel::Build(BrowserWindow
       }
     }
   }
+  if (browser_defaults::kShowExitMenuItem) {
+    AddSeparator(ui::NORMAL_SEPARATOR);
+    AddItemWithStringIdAndVectorIcon(
+        this, IDC_EXIT, IDS_EXIT,
+        features::IsRoundedIconsEnabled() ? kExitToAppIcon : kExitMenuOldIcon);
+  }
 }
 
 ////////////////////////////////////////////////////////////////////////////////
@@ -1940,14 +1824,6 @@ void AppMenuModel::LogMenuMetrics(int co
       }
       LogMenuAction(MENU_ACTION_SAFETY_HUB_MANAGE_EXTENSIONS);
       break;
-    case IDC_TOGGLE_VERTICAL_TABS:
-      if (auto* controller =
-              tabs::VerticalTabStripStateController::From(browser_)) {
-        const bool is_vertical = !controller->ShouldDisplayVerticalTabs();
-        tabs::RecordVerticalTabStripModeChanged(
-            is_vertical, tabs::VerticalTabStripEntryPoint::kAppMenu);
-      }
-      break;
     default: {
       if (IsOtherProfileCommand(command_id)) {
         if (!uma_action_recorded_) {
@@ -2061,24 +1937,8 @@ void AppMenuModel::Build() {
   // Build (and, by extension, Init) should only be called once.
   DCHECK_EQ(0u, GetItemCount());
 
-  if (app_menu_icon_controller_ &&
-      app_menu_icon_controller_->GetTypeAndSeverity().type ==
-          AppMenuIconController::IconType::kUpgradeNotification) {
-    AddSeparator(ui::SPACING_SEPARATOR);
-    const auto update_icon = ui::ImageModel::FromVectorIcon(
-        features::IsRoundedIconsEnabled()
-            ? kRocketLaunchIcon
-            : kBrowserToolsUpdateChromeRefreshOldIcon,
-        ui::kColorMenuIconOnEmphasizedBackground, kDefaultIconSize);
-    if (browser_defaults::kShowUpgradeMenuItem) {
-      AddItemWithIcon(IDC_UPGRADE_DIALOG, GetUpgradeDialogTitleText(),
-                      update_icon);
-      AddSeparator(ui::SPACING_SEPARATOR);
-    }
-  }
-
-  if (AddSafetyHubMenuItem() || AddGlobalErrorMenuItems() ||
-      AddDefaultBrowserMenuItems()) {
+  if (AddGlobalErrorMenuItems() || AddDefaultBrowserMenuItems() ||
+      AddUpgradeMenuItem()) {
     AddSeparator(ui::NORMAL_SEPARATOR);
   }
 
@@ -2120,37 +1980,6 @@ void AppMenuModel::Build() {
 
   AddSeparator(ui::NORMAL_SEPARATOR);
 
-#if !BUILDFLAG(IS_CHROMEOS)
-  sub_menus_.push_back(std::make_unique<ProfileSubMenuModel>(
-      this, browser()->GetProfile(),
-      BrowserWindow::FromBrowser(browser())->GetColorProvider()));
-  auto* const profile_submenu_model =
-      static_cast<ProfileSubMenuModel*>(sub_menus_.back().get());
-  AddSubMenu(kProfileMenuPlaceholder, profile_submenu_model->profile_name(),
-             profile_submenu_model);
-  SetIconForCommandId(kProfileMenuPlaceholder,
-                      profile_submenu_model->avatar_image_model());
-  SetElementIdentifierAt(GetIndexOfCommandId(kProfileMenuPlaceholder).value(),
-                         kProfileMenuItem);
-  SetAccessibleNameAt(GetIndexOfCommandId(kProfileMenuPlaceholder).value(),
-                      profile_submenu_model->accessible_name());
-  AddSeparator(ui::SPACING_SEPARATOR);
-#endif
-
-  if (!browser_->GetProfile()->IsGuestSession()) {
-    sub_menus_.push_back(
-        std::make_unique<PasswordsAndAutofillSubMenuModel>(this));
-    AddSubMenuWithStringIdAndVectorIcon(
-        this, kPasswordsAndAutofillMenuPlaceholder,
-        IDS_PASSWORDS_AND_AUTOFILL_MENU, sub_menus_.back().get(),
-        features::IsRoundedIconsEnabled()
-            ? vector_icons::kPasswordManagerIcon
-            : vector_icons::kPasswordManagerOldIcon);
-    SetElementIdentifierAt(
-        GetIndexOfCommandId(kPasswordsAndAutofillMenuPlaceholder).value(),
-        kPasswordAndAutofillMenuItem);
-  }
-
   if (!browser_->GetProfile()->IsOffTheRecord()) {
     auto recent_tabs_sub_menu =
         std::make_unique<RecentTabsSubMenuModel>(provider_, browser_);
@@ -2175,7 +2004,7 @@ void AppMenuModel::Build() {
         std::make_unique<BookmarkSubMenuModel>(this, browser_);
 
     AddSubMenuWithStringIdAndVectorIcon(
-        this, kBookmarksMenuPlaceholder, IDS_BOOKMARKS_AND_LISTS_MENU,
+        this, kBookmarksMenuPlaceholder, IDS_BOOKMARKS_LIST_TITLE,
         bookmark_sub_menu_model_.get(),
         features::IsRoundedIconsEnabled() ? kStarIcon
                                           : kBookmarksListsMenuOldIcon);
@@ -2197,29 +2026,11 @@ void AppMenuModel::Build() {
         kTabGroupsMenuItem);
   }
 
-  // Extensions sub menu.
-  if (ArePromotionsEnabled() &&
-      base::FeatureList::IsEnabled(features::kExtensionsCollapseMainMenu) &&
-      !extensions::ui_util::HasManageableExtensions(browser_->GetProfile())) {
-    AddItemWithStringIdAndVectorIcon(
-        this, IDC_FIND_EXTENSIONS, IDS_FIND_EXTENSIONS,
-        features::IsRoundedIconsEnabled()
-            ? vector_icons::kChromeExtensionIcon
-            : vector_icons::kExtensionChromeRefreshOldIcon);
-    SetElementIdentifierAt(GetIndexOfCommandId(IDC_FIND_EXTENSIONS).value(),
-                           ExtensionsMenuModel::kVisitChromeWebStoreMenuItem);
-  } else {
-    sub_menus_.push_back(std::make_unique<ExtensionsMenuModel>(this, browser_));
-    AddSubMenuWithStringIdAndVectorIcon(
-        this, kExtensionsSubmenuPlaceholder, IDS_EXTENSIONS_SUBMENU,
-        sub_menus_.back().get(),
-        features::IsRoundedIconsEnabled()
-            ? vector_icons::kChromeExtensionIcon
-            : vector_icons::kExtensionChromeRefreshOldIcon);
-    SetElementIdentifierAt(
-        GetIndexOfCommandId(kExtensionsSubmenuPlaceholder).value(),
-        kExtensionsMenuItem);
-  }
+  AddItemWithStringIdAndVectorIcon(
+      this, IDC_EXTENSIONS_SUBMENU_MANAGE_EXTENSIONS, IDS_EXTENSIONS_SUBMENU,
+      features::IsRoundedIconsEnabled()
+          ? vector_icons::kChromeExtensionIcon
+          : vector_icons::kExtensionChromeRefreshOldIcon);
 
   AddItemWithStringIdAndVectorIcon(
       this, IDC_CLEAR_BROWSING_DATA, IDS_CLEAR_BROWSING_DATA,
@@ -2269,9 +2080,6 @@ void AppMenuModel::Build() {
             lens::features::kLensOverlay));
   }
 
-  AddItemWithStringIdAndVectorIcon(this, IDC_SHOW_TRANSLATE, IDS_SHOW_TRANSLATE,
-                                   vector_icons::kGTranslateIcon);
-
   CreateFindAndEditSubMenu();
 
   sub_menus_.push_back(
@@ -2333,16 +2141,19 @@ void AppMenuModel::Build() {
 #endif
 #endif
 
+  if (!browser_->GetProfile()->IsIncognitoProfile() &&
+      !browser_->GetProfile()->IsGuestSession()) {
+    AddItemWithStringIdAndVectorIcon(this, IDC_SHOW_CUSTOMIZE_CHROME_SIDE_PANEL,
+                                     IDS_SHOW_CUSTOMIZE_CHROME_SIDE_PANEL,
+                                     features::IsRoundedIconsEnabled()
+                                         ? kEditIcon
+                                         : kEditChromeRefreshOldIcon);
+  }
+
   AddItemWithStringIdAndVectorIcon(
       this, IDC_OPTIONS, IDS_SETTINGS,
       features::IsRoundedIconsEnabled() ? kSettingsIcon : kSettingsMenuOldIcon);
 
-  if (browser_defaults::kShowExitMenuItem) {
-    AddItemWithStringIdAndVectorIcon(
-        this, IDC_EXIT, IDS_EXIT,
-        features::IsRoundedIconsEnabled() ? kExitToAppIcon : kExitMenuOldIcon);
-  }
-
   // On Chrome OS, similar UI is displayed in the system tray menu, instead of
   // this menu.
 #if !BUILDFLAG(IS_CHROMEOS)
@@ -2445,35 +2256,23 @@ bool AppMenuModel::AddDefaultBrowserMenu
   return false;
 }
 
-bool AppMenuModel::AddSafetyHubMenuItem() {
-  auto* safety_hub_menu_notification_service =
-      SafetyHubMenuNotificationServiceFactory::GetForProfile(
-          browser_->GetProfile());
-  if (!safety_hub_menu_notification_service) {
-    return false;
-  }
+bool AppMenuModel::AddUpgradeMenuItem() {
+  if (app_menu_icon_controller_ &&
+      app_menu_icon_controller_->GetTypeAndSeverity().type ==
+          AppMenuIconController::IconType::kUpgradeNotification &&
+      browser_defaults::kShowUpgradeMenuItem) {
+    const auto update_icon = ui::ImageModel::FromVectorIcon(
+        features::IsRoundedIconsEnabled()
+            ? kRocketLaunchIcon
+            : kBrowserToolsUpdateChromeRefreshOldIcon,
+        ui::kColorMenuIcon, kDefaultIconSize);
+    AddItemWithIcon(IDC_UPGRADE_DIALOG, GetUpgradeDialogTitleText(),
+                    update_icon);
 
-  std::optional<MenuNotificationEntry> notification =
-      safety_hub_menu_notification_service->GetNotificationToShow();
-  if (!notification.has_value()) {
-    return false;
+    return true;
   }
-  base::UmaHistogramEnumeration("Settings.SafetyHub.Impression",
-                                safety_hub::SafetyHubSurfaces::kThreeDotMenu);
-  base::UmaHistogramEnumeration(
-      "Settings.SafetyHub.EntryPointImpression",
-      safety_hub::SafetyHubEntryPoint::kMenuNotifications);
-  base::UmaHistogramEnumeration("Settings.SafetyHub.MenuNotificationImpression",
-                                notification->module);
-  const auto safety_hub_icon = ui::ImageModel::FromVectorIcon(
-      features::IsRoundedIconsEnabled() ? kSecurityIcon : kSecurityOldIcon,
-      ui::kColorMenuIcon, kDefaultIconSize);
-  AddItemWithIcon(notification->command, notification->label, safety_hub_icon);
-  SetExecuteCallbackAt(
-      GetIndexOfCommandId(notification->command).value(),
-      base::BindRepeating(&AppMenuModel::LogSafetyHubInteractionMetrics,
-                          base::Unretained(this), notification->module));
-  return true;
+
+  return false;
 }
 
 bool AppMenuModel::AddHeliumSchemaItem() {
--- a/chrome/browser/ui/toolbar/app_menu_model.h
+++ b/chrome/browser/ui/toolbar/app_menu_model.h
@@ -334,6 +334,9 @@ class AppMenuModel : public ui::SimpleMe
   // Adds a nag to review Helium services permission changes
   bool AddHeliumSchemaItem();
 
+  // Adds a nag to relaunch the browser after an update
+  bool AddUpgradeMenuItem();
+
   // Adds the Safety Hub menu notifications to the menu. Returns a boolean
   // indicating whether any menu items were added.
   [[nodiscard]] bool AddSafetyHubMenuItem();
