--- a/chrome/browser/ui/views/toolbar/browser_app_menu_button.cc
+++ b/chrome/browser/ui/views/toolbar/browser_app_menu_button.cc
@@ -173,13 +173,8 @@ void BrowserAppMenuButton::OnBoundsChang
 }
 
 void BrowserAppMenuButton::UpdateInkdrop() {
-  if (IsLabelPresentAndVisible()) {
-    ConfigureToolbarInkdropForRefresh2023(this, kColorAppMenuChipInkDropHover,
-                                          kColorAppMenuChipInkDropRipple);
-  } else {
-    ConfigureToolbarInkdropForRefresh2023(this, kColorToolbarInkDropHover,
-                                          kColorToolbarInkDropRipple);
-  }
+  ConfigureToolbarInkdropForRefresh2023(this, kColorToolbarInkDropHover,
+                                        kColorToolbarInkDropRipple);
 }
 
 bool BrowserAppMenuButton::IsLabelPresentAndVisible() const {
@@ -198,14 +193,6 @@ bool BrowserAppMenuButton::IsLabelPresen
 }
 
 SkColor BrowserAppMenuButton::GetForegroundColor(ButtonState state) const {
-  if (IsLabelPresentAndVisible()) {
-    const auto* const color_provider = GetColorProvider();
-    if (type_and_severity_.use_primary_colors) {
-      return color_provider->GetColor(kColorAppMenuExpandedForegroundPrimary);
-    }
-    return color_provider->GetColor(kColorAppMenuExpandedForegroundDefault);
-  }
-
   return ToolbarButton::GetForegroundColor(state);
 }
 
@@ -218,39 +205,20 @@ void BrowserAppMenuButton::UpdateTextAnd
 }
 
 bool BrowserAppMenuButton::ShouldPaintBorder() const {
-  return false;
+  return true;
 }
 
 void BrowserAppMenuButton::UpdateLayoutInsets() {
-  if (IsLabelPresentAndVisible()) {
-    SetLayoutInsets(::GetLayoutInsets(BROWSER_APP_MENU_CHIP_PADDING));
-    SetHorizontalAlignment(gfx::ALIGN_RIGHT);
-  } else {
-    SetLayoutInsets(::GetLayoutInsets(TOOLBAR_BUTTON));
-    SetHorizontalAlignment(gfx::ALIGN_CENTER);
-  }
+  SetLayoutInsets(::GetLayoutInsets(TOOLBAR_BUTTON));
+  SetHorizontalAlignment(gfx::ALIGN_CENTER);
 }
 
 std::optional<SkColor> BrowserAppMenuButton::GetHighlightTextColor() const {
-  if (IsLabelPresentAndVisible()) {
-    const auto* const color_provider = GetColorProvider();
-    if (type_and_severity_.use_primary_colors) {
-      return color_provider->GetColor(kColorAppMenuExpandedForegroundPrimary);
-    }
-    return color_provider->GetColor(kColorAppMenuExpandedForegroundDefault);
-  }
   return std::nullopt;
 }
 
 std::optional<SkColor> BrowserAppMenuButton::GetHighlightColor() const {
-  const auto* const color_provider = GetColorProvider();
-  if (type_and_severity_.severity == AppMenuIconController::Severity::kNone) {
-    return std::nullopt;
-  } else {
-    return color_provider->GetColor(type_and_severity_.use_primary_colors
-                                        ? kColorAppMenuHighlightPrimary
-                                        : kColorAppMenuHighlightDefault);
-  }
+  return std::nullopt;
 }
 
 void BrowserAppMenuButton::OnTouchUiChanged() {
