--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -176,7 +176,7 @@ using PermanentFolderType = BookmarkPare
 bool animations_enabled = true;
 
 // Thickness of the separator (|) in dips (density-independent pixels).
-constexpr int kBookmarkBarSeparatorThickness = 2;
+constexpr int kBookmarkBarSeparatorThickness = 1;
 
 gfx::ImageSkia* GetImageSkiaNamed(int id) {
   return ui::ResourceBundle::GetSharedInstance().GetImageSkiaNamed(id);
@@ -372,8 +372,8 @@ class BookmarkBarView::ButtonSeparatorVi
 
  public:
   ButtonSeparatorView() {
-    const int leading_padding = 8;
-    const int trailing_padding = 8;
+    const int leading_padding = 3;
+    const int trailing_padding = 3;
     const gfx::Insets border_insets =
         gfx::Insets::TLBR(0, leading_padding, 0, trailing_padding);
     const ui::ColorId color_id = kColorBookmarkBarSeparatorChromeRefresh;
@@ -730,7 +730,7 @@ void BookmarkBarView::Layout(PassKey) {
   }
 
   int x = GetLeadingMargin();
-  static constexpr int kBookmarkBarTrailingMargin = 8;
+  static constexpr int kBookmarkBarTrailingMargin = 6;
   int width = View::width() - x - kBookmarkBarTrailingMargin;
 
   const int button_height =
@@ -851,7 +851,7 @@ void BookmarkBarView::Layout(PassKey) {
     // the bookmark buttons.
     if (saved_tab_group_bar_width > 0) {
       // Possibly update the paddings of the separator
-      constexpr int kSeparatorPadding = 8;
+      constexpr int kSeparatorPadding = 6;
       const int left_padding = saved_tab_group_bar_->IsOverflowButtonVisible()
                                    ? 0
                                    : kSeparatorPadding;
