--- a/chrome/browser/resources/new_tab_page_incognito_guest/guest_tab.html
+++ b/chrome/browser/resources/new_tab_page_incognito_guest/guest_tab.html
@@ -10,8 +10,6 @@
 <div class="content">
   <h1>$i18n{guestTabHeading}</h1>
   <p>$i18n{guestTabDescription}</p>
-  <a class="learn-more-button" href="$i18n{learnMoreLink}"
-      aria-label="$i18nPolymer{learnMoreA11yLabel}">$i18n{learnMore}</a>
 </div>
 </body>
 </html>
--- a/chrome/browser/resources/new_tab_page_incognito_guest/incognito_tab.html
+++ b/chrome/browser/resources/new_tab_page_incognito_guest/incognito_tab.html
@@ -29,19 +29,6 @@ document.querySelector('#incognitothemec
     <div class="bulletpoints first">$i18nRaw{incognitoTabFeatures}</div>
     <div class="bulletpoints">$i18nRaw{incognitoTabWarning}</div>
   </div>
-  <div id="cookie-controls">
-    <div id="cookie-controls-description">
-      <div id="cookie-controls-header">
-        <svg viewBox="0 -960 960 960" aria-hidden="true">
-          <path d="m637-425-62-62q4-38-23-65.5T487-576l-62-62q13-5 27-7.5t28-2.5q70 0 119 49t49 119q0 14-2.5 28t-8.5 27Zm133 133-52-52q36-28 65.5-61.5T833-480q-49-101-144.5-158.5T480-696q-26 0-51 3t-49 10l-58-58q38-15 77.5-21t80.5-6q143 0 261.5 77.5T912-480q-22 57-58.5 103.5T770-292Zm-2 202L638-220q-38 14-77.5 21t-80.5 7q-143 0-261.5-77.5T48-480q22-57 58-104t84-85L90-769l51-51 678 679-51 51ZM241-617q-35 28-65 61.5T127-480q49 101 144.5 158.5T480-264q26 0 51-3.5t50-9.5l-45-45q-14 5-28 7.5t-28 2.5q-70 0-119-49t-49-119q0-14 3.5-28t6.5-28l-81-81Zm287 89Zm-96 96Z"/>
-        </svg>
-        <em>$i18n{cookieControlsTitle}</em>
-      </div>
-      <span>$i18nRaw{cookieControlsDescription}</span>
-    </div>
-  </div>
-  <a class="learn-more-button" href="$i18n{learnMoreLink}"
-      aria-label="$i18nPolymer{learnMoreA11yLabel}">$i18n{learnMore}</a>
 </div>
 <script type="module" src="incognito_tab.js"></script>
 <!-- Lazy-load cr_elements to avoid performance penalty introduced by loading Polymer -->
--- a/chrome/browser/ui/webui/ntp/ntp_resource_cache.cc
+++ b/chrome/browser/ui/webui/ntp/ntp_resource_cache.cc
@@ -258,7 +258,7 @@ void NTPResourceCache::CreateNewTabIncog
   DCHECK(profile_->HasAnyOffTheRecordProfile());
 
   replacements["incognitoTabDescription"] =
-      l10n_util::GetStringUTF8(IDS_NEW_TAB_OTR_SUBTITLE_WITH_READING_LIST);
+      l10n_util::GetStringUTF8(IDS_NEW_TAB_OTR_SUBTITLE_HELIUM);
   replacements["incognitoTabHeading"] =
       l10n_util::GetStringUTF8(IDS_NEW_TAB_OTR_TITLE);
   replacements["incognitoTabWarning"] =
--- a/components/new_or_sad_tab_strings.grdp
+++ b/components/new_or_sad_tab_strings.grdp
@@ -158,6 +158,9 @@
           Others who use this device won’t see your activity, so you can browse more privately. This won't change how data is collected by websites you visit and the services they use, including Google. Downloads and bookmarks will be saved.
         </message>
       </if>
+      <message name="IDS_NEW_TAB_OTR_SUBTITLE_HELIUM" desc="Subtitle of the Incognito new tab page, explaining to the user that the Incognito mode hides their browsing activity from other people using the same device, and what data is exempt from it.">
+        Helium will not save anything about your browsing in Incognito, but downloads and bookmarks will stick around. Your browsing won't leave any traces on this device, and other users won't be able to see your activity.
+      </message>
       <message name="IDS_NEW_TAB_OTR_SUBTITLE_WITH_READING_LIST" desc="Subtitle of the Incognito new tab page, explaining to the user that the Incognito mode hides their browsing activity from other people using the same device. The second sentence clarifies that there are two important exceptions from this rule - downloaded files and added bookmarks will be persisted even after the Incognito session is closed." formatter_data="android_java">
         Others who use this device won’t see your activity, so you can browse more privately. This won't change how data is collected by websites you visit and the services they use, including Google. Downloads, bookmarks and reading list items will be saved.
       </message>
