--- a/components/qr_code_generator/bitmap_generator.cc
+++ b/components/qr_code_generator/bitmap_generator.cc
@@ -338,11 +338,13 @@ SkBitmap RenderBitmap(base::span<const u
 }
 
 base::expected<GeneratedCode, Error> GenerateAndPrepareCode(
-    base::span<const uint8_t> data) {
+    base::span<const uint8_t> data,
+    bool has_center_image) {
   // The QR version (i.e. size) must be >= 5 because otherwise the dino
   // painted over the middle covers too much of the code to be decodable.
   constexpr int kMinimumQRVersion = 5;
-  auto qr_result = GenerateCode(data, kMinimumQRVersion);
+  auto qr_result = has_center_image ? GenerateCode(data, kMinimumQRVersion)
+                                    : GenerateCode(data);
   if (!qr_result.has_value()) {
     return base::unexpected(qr_result.error());
   }
@@ -393,7 +395,8 @@ base::expected<SkBitmap, Error> Generate
                                                LocatorStyle locator_style,
                                                CenterImage center_image,
                                                QuietZone quiet_zone) {
-  auto qr_code_result = GenerateAndPrepareCode(data);
+  auto qr_code_result = GenerateAndPrepareCode(
+      data, center_image != CenterImage::kNoCenterImage);
   if (!qr_code_result.has_value()) {
     return base::unexpected(qr_code_result.error());
   }
@@ -409,7 +412,7 @@ base::expected<SkBitmap, Error> Generate
                                                LocatorStyle locator_style,
                                                const SkBitmap& center_image,
                                                QuietZone quiet_zone) {
-  auto qr_code_result = GenerateAndPrepareCode(data);
+  auto qr_code_result = GenerateAndPrepareCode(data, /*has_center_image=*/true);
   if (!qr_code_result.has_value()) {
     return base::unexpected(qr_code_result.error());
   }
--- a/chrome/browser/ui/views/qrcode_generator/qrcode_generator_bubble.cc
+++ b/chrome/browser/ui/views/qrcode_generator/qrcode_generator_bubble.cc
@@ -152,9 +152,9 @@ void QRCodeGeneratorBubble::UpdateQRCont
     qr_code = base::unexpected(qrcode_error_override_.value());
   } else {
     qr_code = qr_code_generator::GenerateImage(
-        base::as_byte_span(input), qr_code_generator::ModuleStyle::kCircles,
-        qr_code_generator::LocatorStyle::kRounded,
-        qr_code_generator::CenterImage::kDino,
+        base::as_byte_span(input), qr_code_generator::ModuleStyle::kSquares,
+        qr_code_generator::LocatorStyle::kSquare,
+        qr_code_generator::CenterImage::kNoCenterImage,
         qr_code_generator::QuietZone::kIncluded);
   }
 
