--- a/chrome/browser/ui/views/frame/layout/browser_view_layout_impl.cc
+++ b/chrome/browser/ui/views/frame/layout/browser_view_layout_impl.cc
@@ -319,7 +319,6 @@ void BrowserViewLayoutImpl::ConfigureTop
     // correctly.
     background->SetVisible(true);
     background->SetPrimaryColor(ui::kColorFrameActive);
-    background->SetCorners(CustomCornersBackground::Corners());
   } else {
     // No need to paint otherwise.
     background->SetVisible(false);
--- a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
+++ b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
@@ -61,15 +61,6 @@ constexpr int kVerticalTabsGrabHandleSiz
 // can take up. This is not the only limit on side panel size.
 constexpr float kMaxContentsHeightSidePanelFraction = 2.f / 3.f;
 
-// How much the vertical tab strip's outline fades out when expanding-on-hover.
-// This is to help it blend better with the drop shadow it acquires - especially
-// in dark mode, where the outline is lighter than the tab strip but the shadow
-// is darker.
-//
-// This is a percentage, with 0.0 meaning no change to the outline, and 1.0
-// meaning the outline disappears completely.
-constexpr double kVerticalTabStripOutlineFadeOnHover = 0.5;
-
 // Increases the leading or trailing exclusion padding to `minimum`.
 void IncreasePaddingToMinimum(BrowserLayoutParams& params, int minimum) {
   // Default to increasing the trailing exclusion padding. On ChromeOS, the
@@ -1337,26 +1328,6 @@ void BrowserViewTabbedLayoutImpl::Config
     background->SetPrimaryColor(CustomCornersBackground::ToolbarTheme());
   }
 
-  // By default, this is just a flat background.
-  CustomCornersBackground::Corners corners;
-
-  // Rounded corners are drawn in vertical tab strip mode when not maximized or
-  // fullscreen.
-  if (layout_data_->tab_strip_type == TabStripType::kVertical &&
-      layout_data_->window_state == WindowState::kNormal) {
-    corners[CornerOrientation::kTopTrailing] =
-        background->GetWindowCorner(/*upper=*/true);
-    const bool vertical_tab_strip_reaches_top =
-        GetVerticalTabStripCollapsedState() !=
-            VerticalTabStripCollapsedState::kCollapsed ||
-        params.leading_exclusion.IsEmpty();
-    if (!vertical_tab_strip_reaches_top) {
-      corners[CornerOrientation::kTopLeading] =
-          background->GetWindowCorner(/*upper=*/true);
-    }
-  }
-
-  background->SetCorners(corners);
 }
 
 void BrowserViewTabbedLayoutImpl::OnLayoutParamsChanged(
@@ -1395,261 +1366,10 @@ void BrowserViewTabbedLayoutImpl::DoPreL
 
 void BrowserViewTabbedLayoutImpl::DoPostLayoutVisualAdjustments(
     const BrowserLayoutParams& params) {
-  // Want to cut the vertical tabstrip and its decorations out of the top
-  // container in transparency mode.
-  CustomCornersBackground::Cutouts top_container_cutout_views;
-  CustomCornersBackground::Cutouts main_background_cutout_views;
-  CustomCornersBackground::Cutouts tab_strip_cutout_views;
   int side_panel_start = layout_data_->revised_params.visual_client_area.x();
 
-  // Set vertical tabstrip corners.
-  CustomCorners::ColorChoiceWithAlpha frame_color(CustomCorners::FrameTheme(),
-                                                  1.0f);
   if (layout_data_->tab_strip_type == TabStripType::kVertical) {
     side_panel_start = views().vertical_tab_strip_region_view->bounds().right();
-
-    // Vertical tabstrip goes all the way to the top of the window if it is not
-    // collapsed or there are no caption buttons on the leading edge.
-    const VerticalTabStripAnimation& animation =
-        layout_data_->vertical_tab_strip_animation;
-    auto* const vertical_tabs_background =
-        views()
-            .vertical_tab_strip_region_view->background()
-            ->AsA<CustomCornersBackground>();
-    CHECK(vertical_tabs_background)
-        << "Expected vertical tab strip to have a CustomCornersBackground.";
-
-    if (in_glass_mode()) {
-      if (!is_fullscreen(layout_data_->window_state)) {
-        frame_color.opacity = 0.0f;
-      }
-      // Use a curve that goes very close to 1 very quickly, but still has a
-      // visible fade. This isn't perfect, but hopefully with glass
-      // expand-on-hover it will improve.
-      auto vertical_tabs_background_color = frame_color;
-      static const double expand_on_hover_opacity =
-          features::kGlassExpandOnHoverOpacity.Get();
-      vertical_tabs_background_color.opacity = static_cast<float>(
-          (1.0 - animation.expand_on_hover_opacity) * frame_color.opacity +
-          animation.expand_on_hover_opacity * expand_on_hover_opacity);
-      vertical_tabs_background->SetPrimaryColor(vertical_tabs_background_color);
-    } else {
-      vertical_tabs_background->SetPrimaryColor(frame_color);
-    }
-
-    // Ensure that corners of the window remain rounded.
-    CustomCornersBackground::Corners vertical_tabs_corners;
-    if (layout_data_->window_state == WindowState::kNormal) {
-      if (animation.top_offset == 0) {
-        vertical_tabs_corners[CornerOrientation::kTopLeading] =
-            vertical_tabs_background->GetWindowCorner(/*upper=*/true);
-      }
-      vertical_tabs_corners[CornerOrientation::kBottomLeading] =
-          vertical_tabs_background->GetWindowCorner(/*upper=*/false);
-    }
-
-    // When the vertical tabs are below the toolbar but next to the bookmarks
-    // bar, draw a curved corner.
-    if (animation.top_corner < 0.0) {
-      vertical_tabs_corners[CornerOrientation::kTopTrailing].type =
-          views().vertical_tab_strip_region_view->is_expanded_on_hover()
-              ? CustomCornersBackground::CornerType::kRounded
-              : CustomCornersBackground::CornerType::kRoundedWithBackground;
-      vertical_tabs_corners[CornerOrientation::kTopTrailing].radius =
-          base::ClampRound(vertical_tabs_background->default_radius() *
-                           -animation.top_corner);
-    }
-
-    // When the vertical tabs are expanded for hover, it may have a concave
-    // corner.
-    double vertical_tabs_bottom_corner_amount = 0.0;
-    int vertical_tabs_bottom_corner_size = 0;
-    if (animation.bottom_corner < 0.0) {
-      vertical_tabs_bottom_corner_amount = -animation.bottom_corner;
-      vertical_tabs_corners[CornerOrientation::kBottomTrailing].type =
-          CustomCornersBackground::CornerType::kRounded;
-      vertical_tabs_bottom_corner_size =
-          base::ClampRound(vertical_tabs_background->default_radius() *
-                           vertical_tabs_bottom_corner_amount);
-      vertical_tabs_corners[CornerOrientation::kBottomTrailing].radius =
-          vertical_tabs_bottom_corner_size;
-    }
-
-    vertical_tabs_background->SetCorners(vertical_tabs_corners);
-
-    // When the organizer panel is animating open or closed and does not appear
-    // elevated, the background of vertical tabs should fade to match the
-    // background color of the panel.
-    if (delegate().IsOrganizerPanelVisible()) {
-      CustomFloatingCorner* const vertical_tabs_top_corner =
-          views().vertical_tab_strip_top_corner;
-      CustomFloatingCorner* const vertical_tabs_bottom_corner =
-          views().vertical_tab_strip_bottom_corner;
-      if (!views().organizer_panel_container->is_elevated()) {
-        auto organizer_panel_reveal_amount =
-            views().organizer_panel_container->GetResizeAnimationValue();
-        CustomCorners::ColorChoiceWithAlpha const fade_background{
-            organizer_panel::kOrganizerPanelBackgroundColor,
-            static_cast<float>(organizer_panel_reveal_amount)};
-        vertical_tabs_background->SetFadeBackground(fade_background);
-        vertical_tabs_top_corner->SetFadeBackground(fade_background);
-        vertical_tabs_bottom_corner->SetFadeBackground(fade_background);
-      } else {
-        vertical_tabs_background->SetFadeBackground(std::nullopt);
-        vertical_tabs_top_corner->SetFadeBackground(std::nullopt);
-        vertical_tabs_bottom_corner->SetFadeBackground(std::nullopt);
-      }
-    }
-
-    // Apply shadow for expand-on-hover.
-    auto* const shadow_frame =
-        views().vertical_tab_strip_region_view->shadow_frame();
-    if (vertical_tabs_bottom_corner_amount > 0.0) {
-      shadow_frame->SetShadowCornerRadius(vertical_tabs_bottom_corner_size);
-      shadow_frame->SetShadowOpacity(vertical_tabs_bottom_corner_amount);
-      shadow_frame->SetShadowVisible(true);
-    } else {
-      shadow_frame->SetShadowVisible(false);
-    }
-
-    CustomCornersBackground::Outline vertical_tabs_outline;
-    vertical_tabs_outline.color = {
-        .color = kColorVerticalTabStripShadow,
-        // Vertical tabs outline fades partially during expand-on-hover to be
-        // replaced with shadow. Note that the glass alpha will affect the
-        // entire paint operation, so no need to re-multiply it in here.
-        .opacity =
-            static_cast<float>(1.0f - kVerticalTabStripOutlineFadeOnHover *
-                                          vertical_tabs_bottom_corner_amount) *
-            frame_color.opacity,
-    };
-    // Vertical tab strip always draws trailing edge.
-    vertical_tabs_outline.trailing = true;
-    // Top edge is drawn when the tabstrip is not flush with the edge of the
-    // screen, or with a visual element that provides a natural border.
-    if (animation.expand_on_hover_width > 0.0 || animation.top_corner < 0.0 ||
-        (animation.top_corner == 0.0 && !params.leading_exclusion.IsEmpty())) {
-      vertical_tabs_outline.top = true;
-    }
-    if (animation.expand_on_hover_width > 0.0) {
-      vertical_tabs_outline.bottom = true;
-    }
-    vertical_tabs_background->SetOutline(vertical_tabs_outline);
-
-    // Cut the vertical tab strip out of the top container.
-    if (!frame_color.is_opaque()) {
-      top_container_cutout_views.push_back(
-          views().vertical_tab_strip_region_view);
-      if (animation.top_corner > 0) {
-        top_container_cutout_views.push_back(
-            views().vertical_tab_strip_top_corner);
-        tab_strip_cutout_views.push_back(views().vertical_tab_strip_top_corner);
-      }
-      main_background_cutout_views = top_container_cutout_views;
-      if (animation.bottom_corner > 0) {
-        main_background_cutout_views.push_back(
-            views().vertical_tab_strip_bottom_corner);
-        tab_strip_cutout_views.push_back(
-            views().vertical_tab_strip_bottom_corner);
-      }
-    }
-
-    // Do corner cutouts and transparency. These updates are needed in glass-
-    // enabled mode even if glass is off for the current window.
-    if (features::IsGlassFrameEnabled()) {
-      views().vertical_tab_strip_top_corner->SetAlpha(frame_color.opacity);
-      views().vertical_tab_strip_bottom_corner->SetAlpha(frame_color.opacity);
-      vertical_tabs_background->SetCutoutFrom(tab_strip_cutout_views);
-
-      // Has to be done after most other adjustments to ensure the correct
-      // outline path.
-      const bool use_blur_background = IsParentedToAndVisible(
-          views().vertical_tab_strip_background_blur_backdrop,
-          views().browser_view);
-      vertical_tabs_background->SetUseBackgroundBlur(use_blur_background);
-      if (use_blur_background) {
-        // The opacity of the backdrop needs to fade over the entire animation
-        // and much more slowly than the opacity of the foreground, or else
-        // there is a flash at the beginning/end. For now, scale with the width.
-        const double blur_background_opacity = animation.expand_on_hover_width;
-        views().vertical_tab_strip_background_blur_backdrop->UpdateGeometry(
-            views().vertical_tab_strip_region_view, blur_background_opacity);
-      }
-    }
-  } else if (layout_data_->tab_strip_type == TabStripType::kHorizontal &&
-             !is_fullscreen(layout_data_->window_state) && in_glass_mode()) {
-    frame_color.opacity = 0.0f;
-  }
-
-  auto* const toolbar_background =
-      views().toolbar->background()->AsA<CustomCornersBackground>();
-  CHECK(toolbar_background)
-      << "Expected toolbar to have a CustomCornersBackground.";
-
-  // Set toolbar corners.
-  CustomCornersBackground::Corners toolbar_corners;
-  switch (layout_data_->tab_strip_type) {
-    case TabStripType::kHorizontal: {
-      const gfx::Rect toolbar_bounds = views().toolbar->GetBoundsInScreen();
-      const gfx::Rect tabstrip_bounds =
-          views().horizontal_tab_strip_region_view->GetBoundsInScreen();
-      if (toolbar_bounds.y() <= tabstrip_bounds.bottom()) {
-        // Trailing curve is always shown for normal horizontal tabstrip when
-        // the two are vertically adjacent.
-        toolbar_corners[CornerOrientation::kTopTrailing].type =
-            CustomCornersBackground::CornerType::kRoundedWithBackground;
-
-        // If there is anything on the leading side or the first tab is not
-        // selected, then the corner radius is shown, otherwise we hide the
-        // corner radius. (Don't show if the left edges don't line up.)
-        if (!delegate().IsActiveTabAtLeadingWindowEdge() &&
-            toolbar_bounds.x() <= tabstrip_bounds.x()) {
-          toolbar_corners[CornerOrientation::kTopLeading].type =
-              CustomCornersBackground::CornerType::kRoundedWithBackground;
-        }
-      }
-      break;
-    }
-    case TabStripType::kVertical: {
-      // Curve trailing corner when it becomes the top trailing corner of the
-      // browser.
-      if (layout_data_->window_state == WindowState::kNormal &&
-          IsParentedTo(views().top_container, views().browser_view) &&
-          params.trailing_exclusion.IsEmpty()) {
-        toolbar_corners[CornerOrientation::kTopTrailing] =
-            toolbar_background->GetWindowCorner(/*upper=*/true);
-      }
-      break;
-    }
-    default:
-      // This can happen in content fullscreen on Mac, but otherwise doesn't
-      // happen.
-      break;
-  }
-  toolbar_background->SetCorners(toolbar_corners);
-
-  auto* const top_container_background =
-      views().top_container->background()->AsA<CustomCornersBackground>();
-  CHECK(top_container_background)
-      << "Expected top container to have a CustomCornersBackground.";
-
-  // Set up top container cutouts. These need to be updated whether or not glass
-  // is active for the current window.
-  if (features::IsGlassFrameEnabled()) {
-    if (!is_fullscreen(layout_data_->window_state)) {
-      toolbar_background->SetCutoutFrom(top_container_cutout_views);
-      // Cut the toolbar corners out of the top container itself.
-      if (in_glass_mode()) {
-        top_container_cutout_views.push_back(
-            CustomCornersBackground::InverseOf(*toolbar_background));
-      }
-      top_container_background->SetCutoutFrom(top_container_cutout_views);
-    } else {
-      toolbar_background->SetCutoutFrom({});
-      top_container_background->SetCutoutFrom({});
-    }
-    toolbar_background->SetCornerColor(frame_color);
-    top_container_background->SetCornerColor(frame_color);
   }
 
   // Clip the side panel so it doesn't run off the edge of the browser or into
@@ -1679,54 +1399,6 @@ void BrowserViewTabbedLayoutImpl::DoPost
       views().side_panel->SetClipToVisibleArea(std::nullopt);
     }
   }
-
-  if (views().main_background_region &&
-      views().main_background_region->GetVisible()) {
-    auto* const main_background = views()
-                                      .main_background_region->background()
-                                      ->AsA<CustomCornersBackground>();
-    CHECK(main_background)
-        << "Expected main background region to have a CustomCornersBackground.";
-
-    // Do the main area cutouts. These adjustments need to be made regardless of
-    // whether the current window is in glass mode.
-    if (features::IsGlassFrameEnabled()) {
-      main_background->SetCutoutFrom(main_background_cutout_views);
-      main_background->SetCornerColor(frame_color);
-    }
-
-    CustomCornersBackground::Corners main_background_corners;
-
-    // Frame-colored corners are shown at the top in horizontal tabstrip mode.
-    // This doesn't apply in fullscreen, as the tabstrip is not in the window.
-    if (layout_data_->tab_strip_type == TabStripType::kHorizontal &&
-        IsParentedTo(views().horizontal_tab_strip_region_view,
-                     views().browser_view)) {
-      // If (due to narrow width) the top container is not laid out in the main
-      // area, it also doesn't get rounded corners.
-      if (views().main_background_region->y() <= views().top_container->y()) {
-        if (!delegate().IsActiveTabAtLeadingWindowEdge()) {
-          main_background_corners[CornerOrientation::kTopLeading].type =
-              CustomCornersBackground::CornerType::kRoundedWithBackground;
-        }
-        main_background_corners[CornerOrientation::kTopTrailing].type =
-            CustomCornersBackground::CornerType::kRoundedWithBackground;
-      }
-    }
-
-    // Need to ensure the bottom of the window is properly rounded for normal
-    // windows.
-    if (layout_data_->window_state == WindowState::kNormal) {
-      if (layout_data_->tab_strip_type != TabStripType::kVertical) {
-        main_background_corners[CornerOrientation::kBottomLeading] =
-            main_background->GetWindowCorner(/*upper=*/false);
-      }
-      main_background_corners[CornerOrientation::kBottomTrailing] =
-          main_background->GetWindowCorner(/*upper=*/false);
-    }
-
-    main_background->SetCorners(main_background_corners);
-  }
 }
 
 void BrowserViewTabbedLayoutImpl::DoPostLayoutCleanup() {
