--- a/chrome/browser/ui/toolbar/bookmark_sub_menu_model.cc
+++ b/chrome/browser/ui/toolbar/bookmark_sub_menu_model.cc
@@ -98,20 +98,6 @@ void BookmarkSubMenuModel::Build(Browser
   AddItemWithStringId(IDC_IMPORT_SETTINGS, IDS_IMPORT_SETTINGS_MENU_LABEL);
 #endif
 
-  AddSeparator(ui::NORMAL_SEPARATOR);
-
-  reading_list_sub_menu_model_ =
-      std::make_unique<ReadingListSubMenuModel>(delegate());
-  AddSubMenuWithStringIdAndIcon(
-      AppMenuModel::kReadingListMenuPlaceholder, IDS_READING_LIST_MENU,
-      reading_list_sub_menu_model_.get(),
-      ui::ImageModel::FromVectorIcon(features::IsRoundedIconsEnabled()
-                                         ? kListAltIcon
-                                         : kReadingListOldIcon));
-  SetElementIdentifierAt(
-      GetIndexOfCommandId(AppMenuModel::kReadingListMenuPlaceholder).value(),
-      kReadingListMenuItem);
-
   auto set_icon = [this](int command_id, const gfx::VectorIcon& vector_icon) {
     auto index = GetIndexOfCommandId(command_id);
     if (index) {
