--- a/chrome/app/settings_strings.grdp
+++ b/chrome/app/settings_strings.grdp
@@ -1867,6 +1867,33 @@
     <message name="IDS_SETTINGS_ENABLE_DO_NOT_TRACK_DIALOG_LEARN_MORE_ACCESSIBILITY_LABEL" desc="The accessibility label for a link to learn more about Do Not Track">
       Learn more about Do Not Track
     </message>
+    <message name="IDS_SETTINGS_GLOBAL_PRIVACY_CONTROL" desc="The label of the toggle to enable or disable sending the Global Privacy Control signal">
+      Global Privacy Control
+    </message>
+    <message name="IDS_SETTINGS_GLOBAL_PRIVACY_CONTROL_DESCRIPTION" desc="Description of the Global Privacy Control toggle">
+      Tell websites to not sell or share your personal data
+    </message>
+    <message name="IDS_SETTINGS_DO_NOT_TRACK" desc="The label of the toggle to enable or disable sending the Do Not Track signal">
+      Send a "Do Not Track" signal with your browsing traffic
+    </message>
+    <message name="IDS_SETTINGS_DO_NOT_TRACK_DESCRIPTION" desc="Description of the Do Not Track toggle">
+      Ask sites to not track you, but they're unlikely to comply
+    </message>
+    <message name="IDS_SETTINGS_NETWORK_AND_SECURITY" desc="Name of the settings row that opens the security settings page">
+      Network and security
+    </message>
+    <message name="IDS_SETTINGS_NETWORK_AND_SECURITY_DESCRIPTION" desc="Description of the settings row that opens the security settings page">
+      Manage how Helium connects to sites, what privacy signals are allowed, and more
+    </message>
+    <message name="IDS_SETTINGS_NETWORK_SECTION_TITLE" desc="Title of the network section on the network and security settings page">
+      Network
+    </message>
+    <message name="IDS_SETTINGS_PRIVACY_SIGNALS_SECTION_TITLE" desc="Title of the privacy signals section on the network and security settings page">
+      Privacy signals
+    </message>
+    <message name="IDS_SETTINGS_PRIVACY_SIGNALS_DESCRIPTION" desc="Description of privacy signals and their potential effect on browser fingerprinting">
+      Privacy signals communicate your preferences, but do not determine how websites will act. Enabling more signals can make your browser easier to fingerprint by giving it a more unique set of characteristics. Global Privacy Control is more likely to be respected than Do Not Track.
+    </message>
     <message name="IDS_SETTINGS_PERMISSIONS_DESCRIPTION" desc="Description of the controls available on the permissions and site content settings page">
       Controls what information sites can use and show (location, camera, pop-ups, and more)
     </message>
@@ -2040,13 +2067,13 @@
       Don't see any warnings before going to insecure sites
     </message>
     <message name="IDS_SETTINGS_HTTPS_FIRST_MODE_ENABLED_STRICT_LABEL" desc="Label for radio button to enable the stricter form of HTTPS-First Mode">
-      Warns you for insecure public &amp; private sites
+      Warn about insecure public and local network sites
     </message>
     <message name="IDS_SETTINGS_HTTPS_FIRST_MODE_ENABLED_STRICT_SUBLABEL" desc="Secondary, continued explanation for the stricter form of HTTPS-First Mode">
       Private sites might include things like your company’s intranet
     </message>
     <message name="IDS_SETTINGS_HTTPS_FIRST_MODE_ENABLED_BALANCED_LABEL" desc="Label for radio button to enable the less-strict form of HTTPS-First Mode">
-      Warns you for insecure public sites
+      Warn about insecure public sites
     </message>
     <message name="IDS_SETTINGS_HTTPS_FIRST_MODE_ENABLED_BALANCED_SUBLABEL" desc="Secondary, continued explanation for the less strict form of HTTPS-First Mode">
       Doesn’t warn you for private sites, such as your company’s intranet
--- a/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
@@ -2507,14 +2507,20 @@ void AddPrivacyStrings(content::WebUIDat
       {"newBadgeLabel", IDS_NEW_BADGE},
       {"privacyPageTitle", IDS_SETTINGS_PRIVACY},
       {"privacyPageMore", IDS_SETTINGS_PRIVACY_MORE},
-      {"doNotTrack", IDS_SETTINGS_ENABLE_DO_NOT_TRACK},
-      {"doNotTrackDialogTitle", IDS_SETTINGS_ENABLE_DO_NOT_TRACK_DIALOG_TITLE},
-      {"doNotTrackDialogMessage", IDS_SETTINGS_ENABLE_DO_NOT_TRACK_DIALOG_TEXT},
-      {"doNotTrackDialogLearnMoreA11yLabel",
-       IDS_SETTINGS_ENABLE_DO_NOT_TRACK_DIALOG_LEARN_MORE_ACCESSIBILITY_LABEL},
+      {"doNotTrack", IDS_SETTINGS_DO_NOT_TRACK},
+      {"doNotTrackDescription", IDS_SETTINGS_DO_NOT_TRACK_DESCRIPTION},
+      {"globalPrivacyControl", IDS_SETTINGS_GLOBAL_PRIVACY_CONTROL},
+      {"globalPrivacyControlDescription",
+       IDS_SETTINGS_GLOBAL_PRIVACY_CONTROL_DESCRIPTION},
+      {"networkSectionTitle", IDS_SETTINGS_NETWORK_SECTION_TITLE},
+      {"privacySignalsSectionTitle",
+       IDS_SETTINGS_PRIVACY_SIGNALS_SECTION_TITLE},
+      {"privacySignalsDescription", IDS_SETTINGS_PRIVACY_SIGNALS_DESCRIPTION},
+      {"securitySectionTitle", IDS_SETTINGS_SECURITY},
       {"siteSettingsSublabel", IDS_SETTINGS_PERMISSIONS_DESCRIPTION_NORMAL},
-      {"securityPageTitle", IDS_SETTINGS_SECURITY},
-      {"securityPageDescription", IDS_SETTINGS_SECURITY_DESCRIPTION},
+      {"securityPageTitle", IDS_SETTINGS_NETWORK_AND_SECURITY},
+      {"securityPageDescription",
+       IDS_SETTINGS_NETWORK_AND_SECURITY_DESCRIPTION},
       {"noiseTitle", IDS_SETTINGS_NOISE},
       {"noiseLinkDescription", IDS_SETTINGS_NOISE_LINK_DESCRIPTION},
       {"noiseToggleDescription", IDS_SETTINGS_NOISE_TOGGLE_DESCRIPTION},
--- a/chrome/browser/resources/settings/privacy_page/cookies_page.html
+++ b/chrome/browser/resources/settings/privacy_page/cookies_page.html
@@ -8,10 +8,6 @@
       padding-bottom: var(--cr-section-vertical-padding);
     }
 
-    #additionalProtections {
-      padding: 16px var(--cr-section-padding) 8px var(--cr-section-padding);
-    }
-
     #defaultHeader {
       padding-top: 0;
       padding-bottom: 6px;
@@ -133,11 +129,6 @@
         </settings-collapse-radio-button>
       </settings-radio-group>
     </div>
-    <h2 id="additionalProtections">
-      $i18n{trackingProtectionAdvancedLabel}
-    </h2>
-    <settings-do-not-track-toggle id="doNotTrack" prefs="{{prefs}}">
-    </settings-do-not-track-toggle>
     <cr-link-row id="siteDataTrigger" class="hr"
         on-click="onSiteDataClick_" label="$i18n{cookiePageAllSitesLink}"
         role-description="$i18n{subpageArrowRoleDescription}">
--- a/chrome/browser/resources/settings/privacy_page/cookies_page.ts
+++ b/chrome/browser/resources/settings/privacy_page/cookies_page.ts
@@ -21,7 +21,6 @@ import '../privacy_icons.html.js';
 import '../settings_page/settings_subpage.js';
 import '../settings_shared.css.js';
 import '../site_settings/site_list.js';
-import './do_not_track_toggle.js';
 
 import {PrefsMixin} from '/shared/settings/prefs/prefs_mixin.js';
 import {I18nMixin} from 'chrome://resources/cr_elements/i18n_mixin.js';
--- a/chrome/browser/resources/settings/privacy_page/do_not_track_toggle.html
+++ b/chrome/browser/resources/settings/privacy_page/do_not_track_toggle.html
@@ -1,39 +1,7 @@
-    <style include="settings-shared">
-      .toggle {
-        padding: 0 var(--cr-section-padding);
-        margin-bottom: var(--cr-section-vertical-padding);
-      }
-    </style>
-    <settings-toggle-button id="toggle" class="toggle"
+    <style include="settings-shared"></style>
+    <settings-toggle-button id="toggle" class="hr"
       label="$i18n{doNotTrack}"
       pref="{{prefs.enable_do_not_track}}"
-      on-settings-boolean-control-change="onToggleChange_"
-      sub-label="$i18n{trackingProtectionDoNotTrackToggleSubLabel}"
-      icon="settings:forward"
-      no-set-pref>
+      on-change="onToggleChange_"
+      sub-label="$i18n{doNotTrackDescription}">
     </settings-toggle-button>
-    <template is="dom-if" if="[[showDialog_]]"
-        on-dom-change="onDomChange_" restamp>
-      <cr-dialog id="confirmDialog"
-          close-text="$i18n{close}" on-cancel="onDialogCancel_"
-          on-close="onDialogClosed_">
-        <div slot="title">$i18n{doNotTrackDialogTitle}</div>
-        <div slot="body">$i18n{doNotTrackDialogMessage}
-          <a href="$i18nRaw{doNotTrackLearnMoreURL}" target="_blank"
-              aria-description="$i18n{opensInNewTab}"
-              aria-label="$i18n{doNotTrackDialogLearnMoreA11yLabel}">
-            $i18n{learnMore}
-          </a>
-        </div>
-        <div slot="button-container">
-          <cr-button class="cancel-button"
-              on-click="onDialogCancel_">
-            $i18n{cancel}
-          </cr-button>
-          <cr-button class="action-button"
-              on-click="onDialogConfirm_">
-            $i18n{confirm}
-          </cr-button>
-        </div>
-      </cr-dialog>
-    </template>
--- a/chrome/browser/resources/settings/privacy_page/do_not_track_toggle.ts
+++ b/chrome/browser/resources/settings/privacy_page/do_not_track_toggle.ts
@@ -2,26 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'chrome://resources/cr_elements/cr_button/cr_button.js';
-import 'chrome://resources/cr_elements/cr_dialog/cr_dialog.js';
 import '../controls/settings_toggle_button.js';
-import '../icons.html.js';
 import '../settings_shared.css.js';
 
-import {focusWithoutInk} from 'chrome://resources/js/focus_without_ink.js';
 import {PolymerElement} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js';
 
-import type {SettingsToggleButtonElement} from '../controls/settings_toggle_button.js';
 import {MetricsBrowserProxyImpl, PrivacyElementInteractions} from '../metrics_browser_proxy.js';
 
 import {getTemplate} from './do_not_track_toggle.html.js';
 
-export interface SettingsDoNotTrackToggleElement {
-  $: {
-    toggle: SettingsToggleButtonElement,
-  };
-}
-
 export class SettingsDoNotTrackToggleElement extends PolymerElement {
   static get is() {
     return 'settings-do-not-track-toggle';
@@ -40,69 +29,14 @@ export class SettingsDoNotTrackToggleEle
         type: Object,
         notify: true,
       },
-
-      showDialog_: {
-        type: Boolean,
-        value: false,
-      },
     };
   }
 
   declare prefs: {enable_do_not_track: chrome.settingsPrivate.PrefObject};
-  declare private showDialog_: boolean;
 
-  private onDomChange_() {
-    if (this.showDialog_) {
-      this.shadowRoot!.querySelector('cr-dialog')!.showModal();
-    }
-  }
-
-  /**
-   * Handles the change event for the do-not-track toggle. Shows a
-   * confirmation dialog when enabling the setting.
-   */
-  private onToggleChange_(event: Event) {
+  private onToggleChange_() {
     MetricsBrowserProxyImpl.getInstance().recordSettingsPageHistogram(
         PrivacyElementInteractions.DO_NOT_TRACK);
-    const target = event.target as SettingsToggleButtonElement;
-    if (!target.checked) {
-      // Always allow disabling the pref.
-      target.sendPrefChange();
-      return;
-    }
-
-    this.showDialog_ = true;
-  }
-
-  private closeDialog_() {
-    this.shadowRoot!.querySelector('cr-dialog')!.close();
-    this.showDialog_ = false;
-  }
-
-  private onDialogClosed_() {
-    focusWithoutInk(this.toggle_);
-  }
-
-  /**
-   * Handles the shared proxy confirmation dialog 'Confirm' button.
-   */
-  private onDialogConfirm_() {
-    this.toggle_.sendPrefChange();
-    this.closeDialog_();
-  }
-
-  /**
-   * Handles the shared proxy confirmation dialog 'Cancel' button or a cancel
-   * event.
-   */
-  private onDialogCancel_() {
-    this.toggle_.resetToPrefValue();
-    this.closeDialog_();
-  }
-
-  private get toggle_(): SettingsToggleButtonElement {
-    return this.shadowRoot!.querySelector<SettingsToggleButtonElement>(
-        '#toggle')!;
   }
 }
 
--- a/chrome/browser/resources/settings/privacy_page/security/secure_dns.html
+++ b/chrome/browser/resources/settings/privacy_page/security/secure_dns.html
@@ -4,13 +4,6 @@
         flex-direction: column;
       }
 
-      /* TODO(crbug.com/349860796): Replace border override by instead removing
-         the "hr" class from the Secure DNS element once the new "Secure
-         connections" settings section ships by default. */
-      :host(.no-hr) settings-toggle-button.hr {
-        border-top: 0;
-      }
-
       #resolverSelect {
         width: inherit;
       }
@@ -37,7 +30,6 @@
 
     <settings-toggle-button
         id="secureDnsToggle"
-        class="hr"
         pref="{{secureDnsToggle_}}"
         label="$i18n{secureDns}"
         sub-label="[[secureDnsDescription_]]"
--- a/chrome/browser/resources/settings/privacy_page/security/security_page.ts
+++ b/chrome/browser/resources/settings/privacy_page/security/security_page.ts
@@ -16,6 +16,7 @@ import '../../icons.html.js';
 import '../../settings_page/settings_subpage.js';
 import '../../settings_shared.css.js';
 import '../../simple_confirmation_dialog.js';
+import '../do_not_track_toggle.js';
 import './secure_dns.js';
 
 import {PrefsMixin} from '/shared/settings/prefs/prefs_mixin.js';
--- a/chrome/browser/resources/settings/privacy_page/security/security_page.html
+++ b/chrome/browser/resources/settings/privacy_page/security/security_page.html
@@ -43,17 +43,27 @@
           var(--cr-radio-button-checked-ripple-color);
     }
 
-    #secureConnectionsSection h2 {
-      padding: 24px var(--cr-section-padding) 8px;
+    #secureConnectionsSection {
+      margin-bottom: var(--cr-section-vertical-padding);
     }
 
     #httpsOnlyModeToggle,
+    #httpsFirstModeToggle,
     #passwordsLeakToggle {
       padding: 0 var(--cr-section-padding);
     }
 
-    #httpsFirstModeToggle {
-      margin-top: 8px;
+    #globalPrivacyControlToggle {
+      padding-inline-end: var(--cr-section-padding);
+      padding-inline-start: var(--cr-section-padding);
+    }
+
+    #privacySignalsHeader {
+      padding-bottom: var(--cr-section-vertical-padding);
+    }
+
+    #httpsOnlyModeToggle {
+      margin-bottom: var(--cr-section-vertical-padding);
     }
 
     #httpsFirstModeRadioGroup {
@@ -73,10 +83,12 @@
   <settings-subpage page-title="$i18n{securityPageTitle}"
       learn-more-url="$i18n{safeBrowsingHelpCenterURL}"
       route-path$="[[routePath]]">
+    <div class="cr-row first">
+      <h2 class="cr-title-text">$i18n{networkSectionTitle}</h2>
+    </div>
     <template is="dom-if" if="[[enableHttpsFirstModeNewSettings_]]" restamp>
       <div id="secureConnectionsSection">
-        <h2 class="cr-title-text">$i18n{secureConnectionsSectionTitle}</h2>
-        <settings-toggle-button id="httpsFirstModeToggle" class="cr-row first"
+        <settings-toggle-button id="httpsFirstModeToggle" class="hr"
             pref="{{prefs.generated.https_first_mode_enabled}}"
             label="$i18n{httpsOnlyModeTitle}"
             sub-label="[[getHttpsFirstModeSubLabel_(
@@ -103,20 +115,14 @@
                   pref="[[prefs.generated.https_first_mode_enabled]]"
                   label="$i18n{httpsFirstModeEnabledStrictLabel}"
                   class="list-item">
-                <div class="cr-secondary-text">
-                  $i18n{httpsFirstModeEnabledStrictSubLabel}
-                </div>
               </controlled-radio-button>
             </settings-radio-group>
           </div>
         </cr-collapse>
       </div>
     </template>
-    <div class="cr-row first">
-      <h2 class="cr-title-text">$i18n{advancedPageTitle}</h2>
-    </div>
     <template is="dom-if" if="[[!enableHttpsFirstModeNewSettings_]]" restamp>
-      <settings-toggle-button id="httpsOnlyModeToggle"
+      <settings-toggle-button id="httpsOnlyModeToggle" class="hr"
           pref="{{prefs.generated.https_first_mode_enabled}}"
           label="$i18n{httpsOnlyModeTitle}"
           sub-label="[[getHttpsFirstModeSubLabel_(
@@ -124,7 +130,8 @@
           numeric-checked-value="[[httpsFirstModeSettingEnum_.ENABLED_FULL]]">
       </settings-toggle-button>
       <template is="dom-if" if="[[showSecureDnsSetting_]]">
-        <settings-secure-dns prefs="{{prefs}}" id="secureDnsSettingOld">
+        <settings-secure-dns prefs="{{prefs}}" id="secureDnsSettingOld"
+            class="hr">
         </settings-secure-dns>
       </template>
     </template>
@@ -137,12 +144,28 @@
     </template>
 <if expr="is_chromeos">
     <template is="dom-if" if="[[showSecureDnsSettingLink_]]">
-        <cr-link-row id="openChromeOSSecureDnsSettings"
+        <cr-link-row id="openChromeOSSecureDnsSettings" class="hr"
             on-click="onOpenChromeOsSecureDnsSettingsClicked_"
             label="$i18n{openChromeOSSecureDnsSettingsLabel}" external>
         </cr-link-row>
     </template>
 </if>
+    <div id="privacySignalsHeader" class="cr-row">
+      <div>
+        <h2 class="cr-title-text">$i18n{privacySignalsSectionTitle}</h2>
+        <div class="cr-secondary-text">$i18n{privacySignalsDescription}</div>
+      </div>
+    </div>
+    <settings-toggle-button id="globalPrivacyControlToggle" class="hr"
+        pref="{{prefs.helium.global_privacy_control}}"
+        label="$i18n{globalPrivacyControl}"
+        sub-label="$i18n{globalPrivacyControlDescription}">
+    </settings-toggle-button>
+    <settings-do-not-track-toggle id="doNotTrack" prefs="{{prefs}}">
+    </settings-do-not-track-toggle>
+    <div class="cr-row">
+      <h2 class="cr-title-text">$i18n{securitySectionTitle}</h2>
+    </div>
     <cr-link-row id="javascriptOptimizerSettingLink" class="hr"
         on-click="onJavascriptOptimizerSettingsClick_"
         label="$i18n{securityJavascriptOptimizerLinkTitle}"
